scripts/deploy.sh

```
@@ -0,0 +1,46 @@
+echo "1. Git repo 확인 및 pull"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)